modules/ROOT/pages/dataweave-selectors.adoc

```diff
@@ -19,7 +19,7 @@ DataWeave function.
 
 | Single-value | `.keyName` | Any type of value that belongs to a matching key
 | Multi-value | `.*keyName` | Array of values of any matching keys
-| Descendants | `..keyName` | Array of values of any matching descendant keys
+| Descendants | `..*keyName` | Array of values of any matching descendant keys
```

Comment:
This is not correct. The correct is`..<selectorExpression>` That is what it should be The `..` selector takes all the child values and applies the selectorExpression to all the children. So `..<keyName>` applies the Singe-Value selector to all each children  `..*<keyName>` applies the Multi-Value selector to all the children and  `..@<keyName>` will apply the attribute selector